### ISSUE TITLE
Adding the training loss to the wandb loss plot

### ISSUE THREAD

**Issue by erinuclkwon** [alan-turing-institute/ice-station-zebra#119]
### Problem / Goal

We currently have a validation loss plot on wandb. It'd be great to add the training loss to the same plot that already tracks validation loss to help model development.

### What does good look like?

- [x] Wandb loss plot for both training and validation loss
